panel/browser-panel-client.cpp

```diff
@@ -186,6 +194,73 @@ bool QCefBrowserClient::OnBeforePopup(
 	return true;
 }
 
+void QCefBrowserClient::OnBeforeContextMenu(CefRefPtr<CefBrowser>,
+					    CefRefPtr<CefFrame>,
+					    CefRefPtr<CefContextMenuParams>,
+					    CefRefPtr<CefMenuModel> model)
+{
+	if (model->IsVisible(MENU_ID_BACK) &&
+	    model->IsVisible(MENU_ID_FORWARD) &&
+	    (!model->IsVisible(MENU_ID_RELOAD) &&
+	     !model->IsVisible(MENU_ID_RELOAD_NOCACHE))) {
+		model->InsertItemAt(2, MENU_ID_RELOAD_NOCACHE, "&Reload");
+		if (model->IsVisible(MENU_ID_PRINT)) {
+			model->Remove(MENU_ID_PRINT);
+		}
```

Comment:
I don't think you need to nest this here if the intent is to _always_ remove the Print menu item.

Also, do we actually need to check if Print is visible?  Does it throw an exception/error if you try to remove print if it's _not_ visible?

Reply:
Done. Even if it doesn't currently throw an exception, I'd rather check before deleting.